terraform/modules/happy-service-eks/variables.tf

```diff
@@ -213,6 +213,11 @@ variable "routing" {
     scheme : optional(string, "HTTP")
     success_codes : optional(string, "200-499")
     service_type : string
+    service_mesh : bool
+    allow_mesh_services : optional(list(object({
+      service : string,
+      stack : string
+    })), null)
```

Comment:
can we make this one object configuration? Something like:

service_mesh: optional(object({
  enabled: optional(bool, false),
  allowed_services: optional(list(object({
      service : string,
      stack : string
    })), null)
})

Reply:
as opposed to two different configuration options for service mesh